### PR TITLE
Graph view

### LINKED ISSUE #17: Displaying associated mileage from MST (all MLB stadiums), DFS (from Fenway Park), and BFS (from Coors Field)
### Description:
As an average American dad who likes to take road trips with his kids to see baseball games, I would like to see multiple different trip types creates by an MST that connects all the MLB stadiums so that I can take the trip and know what I am in for. I would also like to see a DFS (starting at Fenway park) and a BFS (starting at Coors Field)  that follows the shortest distance. From all three of these I want to see the associated mileage.

### Assumptions:
* There exists a weighted graph for containing the stadiums and teams and distances between stadiums
* There exists a way of traversing the graph
* There exists a way to display the associated distances from the MST, DFS, and BFS

### Estimate:
* 8

### Tasks:
1. Query user of what type of traversal they want to do
2. Create a tree from the MST algorithm connecting all the MLB stadium's using Prim's Algorithm or Kruskal's Algorithm if chosen
3. Perform a DFS starting from Fenway park if DFS chosen
4. Perform BFS starting from Coors Field if BFS chosen
5. Display the associated mileage corresponding to the traversal

### Tests:
1. Verify that the output of MST is correct and does not crash
2. Verify that the output of DFS is correct and does not crash
3. Verify that the output of BFS is correct and does not crash

### Definition of Done:
This story is considered done when the user can select one of the three traversals and then receives as output the associated mileage corresponding to their choice.

### PR DESCRIPTION
**Pull request by WakkaTrout**
Closes #17 

### Key features
*  Added the Graph View to the main window
*  Added the map to the graph view
*  Added buttons to Graph View to display the different traversals
*  Added label to Graph View to show total mileage

### Future Improvements
* Maybe add a list of edges (Not in reqs though)

### Notes
This meets requirements 5-7 on the third page of reqs pages

### CODE REVIEW

**Review by Roger-Chavez — APPROVED** [review]
Looks good, good job 👍🏽

File: app/src/windows/mainwindow.ui [hunk 1]
```diff
@@ -88,7 +93,7 @@ QTreeWidget
      <string notr="true"/>
     </property>
     <property name="currentIndex">
-     <number>0</number>
+     <number>2</number>
```
Comment (ChrisRzech) [inline app/src/windows/mainwindow.ui]: Hmmmmmmm

Comment (WakkaTrout) [reply app/src/windows/mainwindow.ui]: uh I will put it back

**Review by ChrisRzech — APPROVED** [review]
👍

**Pull request merged**